bigtable/google/cloud/bigtable/table.py

```diff
@@ -29,6 +29,14 @@
 from google.cloud.bigtable.row_data import PartialRowsData
 
 
+class RowBelongingError(Exception):
+    """Row from another table."""
+
+
+class TooManyMutationsError(Exception):
+    """The number of mutations for bulk request is too big."""
```

Comment:
I disagree; I prefer custom exceptions so the user can be specific about what they want to catch. That said, please subclass `ValueError` rather than `Exception`.